src/xpk/commands/cluster.py

```diff
@@ -423,6 +425,23 @@ def run_gke_cluster_delete_command(args) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
+  if not args.force:
+    xpk_print('Get the name of the workloads in the cluster.')
+    args.filter_by_status = 'EVERYTHING'
+    return_code, return_value = get_workload_list(args)
+    if return_code != 0:
+      xpk_print(f'List Job request returned ERROR {return_code}')
+      return return_code
+
+    workloads = [x.split(' ')[0] for x in return_value.splitlines()][1:]
```

Comment:
what is the output? will workloads be all printed in one line?

Reply:
If so, it would be nice to have them listed in new lines

Reply:
```
Planning to delete 3 workloads in the cluster xpk-test including ['sample-job-kmr9w', 'sample-job-rbkkw', 'sample-job-v7l9r']. 
Do you wish to delete: y (yes) / n (no):
```

That what we have.

Reply:
I think it's okay because the terminal should automatically justify it.